### PR TITLE
Release 14.0.0

### PR DESCRIPTION
**Pull request by vpnguye2**
**Summary:** Summary of changes

 [CUMULUS-3135: Release Cumulus v14.0.0](https://bugs.earthdata.nasa.gov/browse/CUMULUS-3135)

## Changes

* Detailed list or prose of changes
* ...

## PR Checklist

- [ ] Update CHANGELOG
- [ ] Unit tests
- [ ] Ad-hoc testing - Deploy changes and test manually
- [ ] Integration tests

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+## [v14.0.0] 2022-12-08
```
Comment (jennyhliu) [inline CHANGELOG.md:9]: The release v14.0.0 should include everything after https://github.com/nasa/cumulus/releases/tag/v13.4.0.  Please update CHANGELOG.md

File: CHANGELOG.md [hunk 2]
```diff
@@ -86,44 +57,6 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
...
-  - Updated `POST` granules endpoint to check if granuleId exists across all collections rather than a single collection.
-  - Updated `PUT` granules endpoint to check if granuleId exists across a different collection and throw conflict error if so.
-  - Updated logic for writing granules from a message to check if granuleId exists across a different collection and throw conflict error if so.
-
-### Fixed
-
-- **CUMULUS-3104**
-  - Fixed TS compilation error on aws-client package caused by @aws-sdk/client-s3 3.202.0 upgrade
-
-## [v13.4.0] 2022-10-31
```
Comment (jennyhliu) [inline CHANGELOG.md:119]: This section should be untouched.

**Comment by jennyhliu** [comment]
Did you run step 7 and 8?
https://github.com/nasa/cumulus/blob/master/docs/development/release.md#7-update-cumulus-package-api-documentation
https://github.com/nasa/cumulus/blob/master/docs/development/release.md#8-cut-new-version-of-cumulus-documentation

**Comment by jennyhliu** [comment]
Please make sure ci build passes before merging to release branch.

File: CHANGELOG.md (hunk 1 above)
Comment (vpnguye2) [reply CHANGELOG.md:9]: Updated the CHANGELOG.md as commented. Thanks Jenny.

File: CHANGELOG.md (hunk 2 above)
Comment (vpnguye2) [reply CHANGELOG.md:119]: Updated the CHANGELOG.md as commented

**Comment by vpnguye2** [comment]
> Did you run step 7 and 8?
> https://github.com/nasa/cumulus/blob/master/docs/development/release.md#7-update-cumulus-package-api-documentation
> https://github.com/nasa/cumulus/blob/master/docs/development/release.md#8-cut-new-version-of-cumulus-documentation

For step 8, Do I need to change any version in the website/package.json before running "npm run version v1.4.0"? Thanks Jenny

File: CHANGELOG.md [hunk 3]
```diff
@@ -595,37 +550,6 @@ releases.
     `example/cumulus-tf/sync_granule_workflow.asl.json` to include `ACL`
     parameter.
 
-## [v11.1.8] 2022-11-07 [BACKPORT]
```
Comment (jennyhliu) [inline CHANGELOG.md:598]: I'm confused.  Why is this deleted?  Any thing under ## [v13.4.0] 2022-10-31 shouldn't be changed
